1.0/nanoserver/runtime/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# escape=`
+FROM microsoft/dotnet:1.0.4-runtime-nanoserver
```

Comment:
The Windows Container team has asked that we produce nanoserver versioned tags for the time being (e.g microsoft/dotnet:1.0.4-sdk-nanoserver-10.0.14393.953). Because of that ask, I think you would need to reference a nanoserver version tag.  If you don't, your versioned tags (e.g. ...nanoserver-10.0.14393.953) will get updated with the vnext nanoserver bits whenever they become available.